### ISSUE TITLE
When I test a single image. I always have an ERROR: error code -6

### ISSUE THREAD

**Issue by howtocodewang** [NVIDIA/DIGITS#1306]
OS: Ubuntu 14.04
GPU: K40c
Digits: 5.1-dev
Caffe:0.15.14

I want to Classify One Image using my model. I have never succeeded with a problem.

In the Job blog, I can see this:

WARNING: Logging before InitGoogleLogging() is written to STDERR
F1129 17:14:32.927536 13462 gpu_memory.cpp:168] Check failed: error == cudaSuccess (46 vs. 0)  all CUDA-capable devices are busy or unavailable
*** Check failure stack trace: ***

But I cannot solve this. If I use torch7 instead of caffe, I can classify the image. I don't know why and how to solve.

**Comment by lukeyeager** [comment]
Probably a duplicate of https://github.com/NVIDIA/DIGITS/issues/1260. Does the discussion there help you?

**Comment by howtocodewang** [comment]
Thank you for your reply.

I have tried the method you mentioned in [#1260](url) and finally it works very perfectly. Thank you very much!!

By the way, I want to know if the DIGITS 5 which will be released in December would be installed on Ubuntu 16.04LTS as easily as on Ubuntu 14.04LTS. 

I tried to build DIGITS from source following [https://github.com/NVIDIA/DIGITS/blob/digits-4.0/docs/BuildDigits.md#building-digits](url) on my laptop that is using Ubuntu 16.04LTS, but I failed many times.
